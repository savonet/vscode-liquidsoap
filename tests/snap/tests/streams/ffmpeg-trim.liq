def f() =
  def on_error(_) =
    test.fail()
  end

  # Test string.trim on url input.
  output.dummy(
    input.ffmpeg(on_connect=test.pass, on_error=on_error, " ./file1.mp3\n"),
    fallible=true
  )
end

test.check(f)
