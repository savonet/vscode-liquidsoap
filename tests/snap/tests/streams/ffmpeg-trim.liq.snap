>def f() =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^ source.liquidsoap
>  def on_error(_) =
#^^ source.liquidsoap
#  ^^^ source.liquidsoap keyword.control.liquidsoap
#     ^^^^^^^^^^^^^^^ source.liquidsoap
>    test.fail()
#^^^^^^^^^^^^^^^^ source.liquidsoap
>  end
#^^ source.liquidsoap
#  ^^^ source.liquidsoap keyword.control.liquidsoap
>
>  # Test string.trim on url input.
#^^ source.liquidsoap
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>  output.dummy(
#^^^^^^^^^^^^^^^^ source.liquidsoap
>    input.ffmpeg(on_connect=test.pass, on_error=on_error, " ./file1.mp3\n"),
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
#                                                          ^ source.liquidsoap string.quoted.double.liquidsoap
#                                                           ^^^^^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                                                                       ^^ source.liquidsoap string.quoted.double.liquidsoap constant.character.escape.liquidsoap
#                                                                         ^ source.liquidsoap string.quoted.double.liquidsoap
#                                                                          ^^^ source.liquidsoap
>    fallible=true
#^^^^^^^^^^^^^^^^^^ source.liquidsoap
>  )
#^^^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>test.check(f)
#^^^^^^^^^^^^^^ source.liquidsoap
>